### PR TITLE
[AE-39] Add API documentation to Camera library

### PR DESCRIPTION
**Pull request by sebromero**
This PR adds the missing documentation to the Camera library. Currently the API documentation is rendered by hand. This process will be automated with a future PR.

### CODE REVIEW

File: libraries/Camera/docs/README.md
Comment (alrvid) [inline libraries/Camera/docs/README.md:1]: Should the Himax HM01B0 be mentioned too in the list of compatible camera models?

File: libraries/Camera/library.properties
[the remark this answers is not in the document]
Comment (alrvid) [reply libraries/Camera/library.properties:1]: Should the fourth camera model be mentioned here too?

File: libraries/Camera/src/camera.h
Comment (alrvid) [inline libraries/Camera/src/camera.h:1]: Should the fourth camera model be mentioned?

Comment (alrvid) [reply libraries/Camera/src/camera.h:1]: Misspelling on line 87.

Comment (alrvid) [reply libraries/Camera/src/camera.h:1]: The reset() function isn't specifically for getting out of stuck states. What it does differs between the camera models. It does nothing at all on the GC2145. On the OV7670, it reset all configuration registers to the default values. The registers are reset to the defaults on the Himax cameras too, but they also enter Standby Mode, where there is no video output and the power consumption is lowered.

File: libraries/Camera/src/camera.h [hunk 1]
```diff
@@ -14,83 +14,315 @@
...
+
+        /**
+         * @brief Output debug information to a stream.
+         * You can use this function to output debug information to the serial port by passing Serial as the stream.
+         * @param stream Stream to output the debug information
+         */
         virtual void debug(Stream &stream) = 0;
 
+        /**
+         * @brief Set the sensor in standby mode.
```
Comment (alrvid) [inline libraries/Camera/src/camera.h:286]: This actually has no effect on none of the currently supported cameras. The Himax cameras can be set in standby mode by calling reset() instead, but not by calling this function.

**Review by alrvid — CHANGES_REQUESTED** [review]
See the comments for each file.

File: libraries/Camera/docs/README.md
Comment (sebromero) [reply libraries/Camera/docs/README.md:1]: Yes

**Pull request merged**